src/components/AddBook.js

```diff
@@ -25,11 +25,11 @@ const AddBook = () => {
     }
   };
   return (
-    <section>
+    <section className="addBook">
       <h2>Add a new book</h2>
       <form onSubmit={handleSubmit}>
-        <input type="text" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Title" />
-        <input type="text" value={author} onChange={(e) => setAuthor(e.target.value)} placeholder="Author" />
+        <input className="formTitle" type="text" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Book title" />
+        <input className="formAuthor" type="drop" value={author} onChange={(e) => setAuthor(e.target.value)} placeholder="Author" />
         <button type="submit">Add Book</button>
```

Comment:
- [ ] As mentioned by the previous reviewer the text of this button is in uppercase as you can see in the design, please kindly fix that 🙏🏾 

Your design

<img width="195" alt="Screenshot 2023-06-17 at 17 27 57" src="https://github.com/lorenzoworx/bookstore-react/assets/74501523/f1f521a0-e1d7-49a9-8318-dabe39c587f6">

Given design

<img width="195" alt="Screenshot 2023-06-17 at 17 28 10" src="https://github.com/lorenzoworx/bookstore-react/assets/74501523/95da23fc-1884-4a1e-b14f-e085b15a2360">